test/user.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+
+jest.mock('../src/backend/utils/elastic');
+
+// Mock the internal authentication strategy
+jest.mock('../src/backend/web/authentication');
+// Use our authentication test helper
+const { login, logout } = require('./lib/authentication');
+
+describe.skip('test GET /user/feeds endpoint', () => {
+  let user;
+
+  beforeAll(() => {
+    user = login('Johannes Kepler', 'user1@example.com');
+  });
+
+  beforeEach(() => {
+    logout();
+    login('Johannes Kepler', 'user1@example.com');
+  });
+
+  afterAll(() => logout());
+
+  it('should respond with a 403 status when not logged in', async () => {
+    logout();
+    const res = await request(app).get(`/user/feeds`);
+    expect(res.status).toEqual(403);
+  });
+
+  it('should respond with a 200 status and JSON array if logged in', async () => {
+    const res = await request(app).get(`/user/feeds`);
```

Comment:
Similar to what you did on line 27, I would be explicit and login here, and you can probably get rid of the global `user` variable (i.e, have each test do the login/logout it needs).  NOTE: these `it()` tests will not run in parallel, so no issue of shared state or races.

```js
const user = login('Johannes Kepler', 'user1@example.com');
```

Reply:
Fixed in Fixed in https://github.com/Seneca-CDOT/telescope/pull/958/commits/5cee6b4c37a89f88fa372ea01b50ff1e5cf45b72